test/comparison_to_analytic_solutions.jl

```diff
@@ -19,7 +19,7 @@ e = SolidStateDetectors.elementary_charge * u"C"
     W_true = uconvert(u"J", (ϵr * ϵ0 * E_true^2 / 2) * A * Δd)
     W_ssd = SolidStateDetectors.calculate_stored_energy(sim);
     C_true = uconvert(u"pF", 2 * W_true / (BV_true^2))
-    C_ssd = SolidStateDetectors.calculate_capacitance(sim) 
+    C_ssd = SolidStateDetectors.calculate_capacitance(sim, ElectricPotential, consider_multiplicity = false)
```

Comment:
Because this detector have a multiplicity of infinity in principle.
I defined 
```julia
function multiplicity(g::DiscreteAxis{T, :reflecting, :reflecting, I}, ::Type{Cartesian}) where {T, I} 
    @warn "Multiplicity of Cartesian axis, $(g) (:reflecting, :reflecting), would be infinite. It is set to 1 here." 
    one(T)
end
```
so it works also with `consider_multiplicity = true`. But this is in principle not true as it should be `Inf.

Reply:
Are we expecting `DiscreteAxis{T, :reflecting, :reflecting, I}` in `Cartesian` coordinates?
I think that we only have this in our test case...

Reply:
There might be other cases. E.g., if one wants to simulate a detector in the middle of a detector string.